### ISSUE TITLE
Is the `-e` flag required? If so, it's missing from at least the RPM /etc/init.d/cassandra script.

### ISSUE THREAD

**Issue by AlexisWilke** [strapdata/elassandra#258]
**Elassandra version**: elassandra-6.2.3.8.rpm

**Plugins installed**: [none]

**JVM version** (`java -version`):

> java version "1.8.0_192"
> Java(TM) SE Runtime Environment (build 1.8.0_192-b12)
> Java HotSpot(TM) 64-Bit Server VM (build 25.192-b12, mixed mode)

**OS version** (`uname -a` if on a Unix-like system):

> Linux ip-172-31-24-11.ec2.internal 3.10.0-862.3.2.el7.x86_64 #1 SMP Mon May 21 23:36:36 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

**Description of the problem including expected versus actual behavior**:

The [README.md](https://github.com/strapdata/elassandra#installation) says:

Run `bin/cassandra -e`

The [RPM](https://github.com/strapdata/elassandra/blob/v6.2.3-strapdata/distribution/rpm/src/main/packaging/init.d/cassandra) says:

    daemon --user $CASSANDRA_USER --pidfile $pidfile $exec -p $pidfile

I do not see the `-e` option. Is one of the environment variables enough to make it all work properly?

**Steps to reproduce**:

[Look at the file on line 145](https://github.com/strapdata/elassandra/blob/v6.2.3-strapdata/distribution/rpm/src/main/packaging/init.d/cassandra#L145).

**Please provide the following information:**

* elassandra logs (logs/system.logs or /var/lib/cassandra/system.log)

No log file is generated.

* elasticsearch cluster state (curl http://localhost:9200/_cluster/state)

It doesn't start. But that's probably because of something else.

* cassandra schema (cqlsh>DESC KEYSPACE <your_keyspace>)

No schema since it didn't start.

* cassandra gossip state (run: nodetool gossipinfo)

No gossip since it didn't start.

**Comment by DBarthe** [comment]
Hi @AlexisWilke,

The `init.d/cassandra` is a fallback in case systemd is not installed. It is really not battle-tested. 

The `-e` option is a command line switch to enable elasticsearch at startup. It takes precedence over the  `CASSANDRA_DAEMON` environment variable which indicates the java main class. Since `init.d/cassandra` loads variables from `/etc/sysconfig/cassandra` it is not necessary to add the `-e` switch there.

What command did you use to start elassandra ? If systemd is available, try `systemctl start cassandra` as root.

**Comment by AlexisWilke** [comment]
I used `systemctl start cassandra` and it seems to work fine. The computer is in maintenance right now so I can't check more at this point, but if the `/etc/sysconfig/cassandra` takes care of it, I guess we're all good. I read that init script to make sure I was not missing anything in my setup and that's how I noticed that the `-e` was not there...

On my development machine, though, I just start/stop using my own script and use the `-e` flag.
